BIDS-Dependencies-List.md

```diff
@@ -0,0 +1,31 @@
+# BIDS Dependencies List
+The following is a list of **options** available for working with BIDS
+
+You **do not** need all these programs to work with BIDS. Choose whichever ones you are most comfortable with
```

Comment:
But you do need some of them together, right? So if you're working in matlab you need *either* matlab *or* octave but you *definitely* need JSONio? And you don't have to work in matlab and python and R, but you do need to choose one?

I think this file could be bulked up with a couple more sentences to make it really clear